industries/views.py

```
@@ -283,141 +292,115 @@ def get_industry_news(request, industry_id):
-    try:
-        service = IndustryOutlookService()
-        result = service.analyze_outlook(
-            industry=industry,
-            days_back=days_back,
-            max_news=max_news,
-            max_reports=max_reports,
-            top_companies=top_companies,
-        )
+class IndustryBackfillView(APIView):
+    # 테스트 편의를 위해 인증/권한 일시 해제
+    authentication_classes = [] 
```

Ruff (0.14.13)

Redefinition of unused `authentication_classes` from line 2: `authentication_classes` redefined here

(F811)